kbase-extension/static/kbase/js/widgets/appWidgets2/input/textInput.js

```diff
@@ -4,34 +4,31 @@ define([
     'widgets/appWidgets2/validation',
     'common/events',
     'common/ui',
-    'common/props',
     'common/runtime',
     '../inputUtils',
     'widgets/appWidgets2/validators/constants',
     'bootstrap',
-], (Promise, html, Validation, Events, UI, Props, Runtime, inputUtils, Constants) => {
+], (Promise, html, Validation, Events, UI, Runtime, inputUtils, Constants) => {
     'use strict';
 
     const t = html.tag,
         div = t('div'),
-        input = t('input'),
-        model = Props.make({
```

Comment:
This being declared outside of the factory is what's causing the problems. That makes it apparently global to all text input widgets. Oops!